common.js

```diff
@@ -2802,6 +2802,11 @@ const allSoloCars = {
       '2006': ['fs'],
     },
   },
+  'Jensen': {
+    'Jensen-Healey' : {
+      'all': ['cs', 'fsp', 'pd'] 
+    }
+  }
```

Comment:
Missing a comma here which creates a syntax error.

Also please be sure to add dangling commas everywhere according to this https://eslint.org/docs/rules/comma-dangle#always-multiline

Reply:
Oh , I missed it will fix